web/src/components/tasks/environment-questionnaire/CheckWasteResultsAccordion.tsx

```diff
@@ -0,0 +1,44 @@
+import { Heading } from "@/components/njwds-extended/Heading";
+import { Icon } from "@/components/njwds/Icon";
+import { SectionAccordionContext } from "@/contexts/sectionAccordionContext";
+import { Accordion, AccordionDetails, AccordionSummary } from "@mui/material";
+import { ReactElement, ReactNode, useState } from "react";
+
+interface Props {
+  title: string;
+  children: ReactNode;
+  mini?: boolean;
+  isDividerDisabled?: boolean;
+}
+
+export const CheckWasteResultsAccordion = (props: Props): ReactElement => {
```

Comment:
Does this element need to be specific to waste results instead of a more generic `ResultsAccordion` or something like that?

Reply:
I left it as CheckWaste for now because only waste uses it, but I figured it would make sense to refactor it in a later ticket.